lib/commandhandlers.go

```diff
@@ -97,6 +98,45 @@ func loadClientCommands() {
 		minParams:    1,
 		handler: func(listener *Listener, msg ircmsg.IrcMessage) {
 			// We're starting CAP negotiations so don't complete regisration until then
+			listener.regLocks.Set("cap", false)
+
+			command := strings.ToUpper(getParam(&msg, 0))
+			if command == "LS" {
+				capList := Capabilities.AsString()
+				listener.Send(nil, "", "CAP", "*", "LS", capList)
+
+			} else if command == "REQ" {
```

Comment:
Hmm, would we want to support `-` REQ'ing capabilities here as well? Not sure if there's much of a reason to right now, we can add it later when we have caps that benefit from that.